test/functionalTest/cases/0673_xauth_token/auth_token_from_subscribe_request.test

```diff
@@ -55,7 +55,6 @@ payload='{
   ],
```

Comment:
This test has nothing to do with throttling. I guess it was included due to copy-paste from other tests...

After fixing the throttling issue mentioned in https://github.com/telefonicaid/fiware-orion/pull/3987/files#r753155066 the previous expectation is no longer working as this tests was using throttling 1 second.

NTC (informative)